include/pa_win_wasapi.h

```diff
@@ -82,7 +82,13 @@ typedef enum PaWasapiFlags
        playback formats that do not match the current configured system settings.
        this is in particular required for streams not matching the system mixer sample rate.
        only applies in Shared mode. */
-    paWinWasapiAutoConvert              = (1 << 6)
+    paWinWasapiAutoConvert              = (1 << 6),
+
+    /* force data output to match AC3 passthrough mode */
+    paWinWasapiMatchAc3                 = (1 << 7),
```

Comment:
I think it is worthwhile to make the naming used in the public API to this functionality as nice as possible. Is there a reason to use the verb "match" here? It doesn't seem right to me. Please consider options like:

- `paWinWasapiAc3` and `paWinWasapiEac3`
- `paWinWasapiForceAc3` and `paWinWasapiForceEac3`
- `paWinWasapiFormatAc3` and `paWinWasapiFormatEac3`
- `paWinWasapiAc3Passthrough` and `paWinWasapiEac3Passthrough`

Or maybe something else. You guys are the most knowledgeable and in the best position to decide what captures the meaning best, but I think that I prefer `paWinWasapiAc3` or `paWinWasapiAc3Passthrough`.

Reply:
I prefer `*Passthrough` variants.